platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/MultiWebSocket.js

```diff
@@ -34,23 +34,23 @@ export default class MultiWebSocket {
     this._socketTask = socketTask;
 
     socketTask.onClose((res) => {
-      this.readyState = WebSocket.CLOSED
+      this.readyState = MultiWebSocket.CLOSED
       if (typeof this.onclose === 'function') {
         this.onclose(res)
       }
     })
 
     socketTask.onMessage((res) => {
       if (typeof this.onmessage === 'function') {
-        if (res && res.data && res.isBuffer) {
-          res.data = _utils.base64ToArrayBuffer(res.data)
+        if (res && res.data && res.data.data && res.data.isBuffer) {
```

Comment:
Due to the single-instance version of the WebSocket, the onMessage response res contains only one layer of data. For compatibility with the online game, Taobao removed the extra layer of data. This issue will be corrected in future versions.

Reply:
res.data.data removed at this [PR](https://github.com/cocos/cocos-engine/pull/17389)